src/components/UnreadActionIndicator.js

```diff
@@ -10,7 +10,7 @@ function UnreadActionIndicator(props) {
         <View
             accessibilityLabel={props.translate('accessibilityHints.newMessageLineIndicator')}
             data-action-id={props.reportActionID}
-            style={[styles.unreadIndicatorContainer, styles.userSelectNone]}
+            style={[styles.unreadIndicatorContainer, styles.userSelectNone, styles.pointerEventsNone]}
```

Comment:
NAB. Can we use the pointerEvents prop instead?

Reply:
@s77rt @tienifr `pointerEvents` prop is deprecated and is there any issue using the styles?

Reply:
>No problem in using the styles.

Thanks. 

> Is there a link for this prop being deprecated?

deprecated in rnw check the notes here: https://github.com/necolas/react-native-web/releases

Reply:
Thanks for the link! I guess we can keep this for now, we are still on version 18. Once upgraded that should be handled there.